src/app/pages/adoption-form/adoption-form.js

```diff
@@ -71,5 +71,4 @@ export default class AdoptionForm extends BaseView {
         this.regions.primaryBook.append(bookInfoView);
         this.toggler.update();
     }
-
```

Comment:
I think this blank line was actually intended.  We seem to be leaving a blank line at the start and end of classes.